data/hai/hai missions.txt

```diff
@@ -1932,3 +1932,103 @@ mission "Station Exploration Abandon Ship"
+					goto ask
+					to display
+						or
+							has "visited system: Wah Ki"
+							has "Wanderers: Translation Machine: offered"
+							has "First Contact: Unfettered: offered"
+				`	"I'm not going there. You'll have to ask someone else."`
+					decline
+			label ask
+			`	The Hai pauses for a moment, as if unsure of what to say. "My teachings and prophecies are more... appreciated among our northern brethren."`
+			choice
+				`	"Teachings?"`
+				`	"Alright, I'll take you there."`
+					goto agree
+				`	"Unfettered space is too dangerous for me. You'll have to find someone else."`
+					goto dangerous
+			label teachings
+			`	An unreadable expression appears on Hareek's face. "Ah, apologies for my rudeness - you probably do not know who I am. How do I explain this... You have your preachers and wise ones among the humans, yes? I am something similar. Now, will you take me to <destination>?"`
+			choice
+				`	"So you're going to preach to the Unfettered?"`
+				`	"I can take you."`
+					goto agree
+				`	"Unfettered space is too dangerous for me. You'll have to find someone else."`
+					goto dangerous
+			`	"That is an oversimplification, but it is not something I can explain adequately to a human." He turns away, looking into the fog. "There are portentous events occurring in the galaxy, and I must warn others of the turmoil I have foreseen. The Unfettered crave turmoil, so naturally they are more inclined to believe me." He gives a wry smile.`
+			choice
+				`	"Alright, I can take you there."`
+					goto agree
+				`	"Sorry, but this whole thing sounds a little too strange for me."`
+					goto strange
+				`	"Unfettered space is too dangerous for me. You'll have to find someone else."`
+			label dangerous
+			`	He snorts. "Dangerous? I am approaching 300 years of age, and yet I do not fear a trip to Unfettered space." He walks away, clearly disappointed.`
+				decline
+			label strange
+			`	He sighs. "That's alright. I would not expect you to understand." He walks away, disappearing into the mist.`
+				decline
+			label agree
+			`	"Excellent. Arriving in a human ship would be an auspicious start to my journey, but there are not many human captains who take proper precautions when traveling through space. I'm glad to see that you have some sense, at least." He lowers his voice.`
+			`	"Since you seem to be a trustworthy captain, I will tell you of some things I have seen. Many years ago, I defended our space against the Korath raids. One day, we were caught unprepared and hit by a massive fleet. Every single ship in my patrol group was destroyed, except for ours - the only one with a keystone. I learned two things in my time with the Korath: that the old legends are true, and that the stability we have enjoyed as a species is ending. Changes in the galaxy are inevitable, and those changes are arriving soon." He begins to move toward your ship. "But let us not speak of this now. I have business to attend to."`
```

Comment:
```suggestion
			`	"Since you seem to be a trustworthy captain, I will tell you of some things I have seen. Many years ago, I defended our space against the Korath raids. One day, we were caught unprepared and hit by a massive fleet. Every single ship in my patrol group was destroyed, except for ours - the only one with a keystone. I learned two things in my time against the Korath: that the old legends are true, and that the stability we have enjoyed as a species is ending. Changes in the galaxy are inevitable, and those changes are arriving soon." He begins to move toward your ship. "But let us not speak of this now. I have business to attend to."`
```
`with` almost implied Hareek was with the Korath, not fighting against them. This word changes clarifies the intended meaning.